api/controllers/fileUploader.py

```diff
@@ -136,3 +137,34 @@ def upload_default():
     file_upload = File(filename=imageName, filetype='image', uploader=fetch_user)
     file_upload.save_to_db()
     return jsonify(DefImageSchem().dump(file_upload).data)
+
+
+@router.route('/background_color', methods=['POST'])
+def background_color():
+    try:
+        data = request.get_json()['data']
```

Comment:
Can you please convert it to `request.get_json()['data']['attributes']` else all look fine to me